src/test/java/com/thebois/models/world/structures/StructureTests.java

```diff
@@ -37,9 +51,10 @@ public void structureEqualItSelfIsTrue() {
     @Test
     public void structureEqualIsTrueIfSamePositionAndType() {
         // Arrange
-        final Position position = new Position(1, 1);
-        final AbstractStructure structure1 = new House(position);
-        final AbstractStructure structure2 = new House(position);
+        final Position position1 = new Position(1, 1);
+        final Position position2 = new Position(1, 1);
+        final AbstractStructure structure1 = new House(position1);
+        final AbstractStructure structure2 = new House(position2);
```

Comment:
style

Why did you create a duplicate position when the test is supposed to be using the same position for both structures?